packages/fast-jss-utilities/README.md

```diff
@@ -28,6 +28,43 @@ const styles = {
 };
 ```
 
+### applyFocusVisible
+`applyFocusVisible` accepts style(s) and an optional selector string. `applyFocusVisible` automatically removes user-agent focus outlines by setting `outline: "none"`.
+
+This function implements the utility for all focus states, if you want to enable focus-visible support you should implement the focus-visible polyfill. Styles that are given will be returned with a focus selector that is based on whether there is a focus polyfill for focus-visible. If a selector has been passed, this will be added to the focus selector string for the styles. If you are padding a selector be sure to include a space before the selector so it appends properly.

```

Comment:
minor nit here, we actually don't test for the polyfill, we instead test for native support of `:focus-visible` - if the browser doesn't support `:focus-visible` natively, we apply selectors that align to the polyfill's behavior.